tests/utils.test.js

```diff
@@ -98,6 +99,19 @@ describe('isTimeToLiveExpired', () => {
   });
 });
 
+describe('parseOptions', () => {
+  const timeToLive = new Date().getTime();
+
+  test('Sets \'timeToLive\' to zero if none is given', () => {
+    expect(parseOptions({})).toEqual({ timeToLive: 0 });
+  });
+
+  test('Parses the options correctly', () => {
+    expect(parseOptions({ timeToLive }))
+      .toEqual({ timeToLive: calculateExpiry(timeToLive) });
```

Comment:
I am not quite happy with this _expectation_. I feel like I am just rebuilding the functionality here. Not really sure what to do about this, maybe we can discuss?